src/components/Attachment/Gallery.js

```diff
@@ -1,294 +1,240 @@
-    }
+  const galleryImages = [...images].map((i) => ({
+    url: makeImageCompatibleUrl(i.image_url || i.thumb_url),
+  }));
 
+  if (galleryImages.length === 1) {
     return (
-      <React.Fragment>
-        <GalleryContainer
-          length={images.length}
-          alignment={this.props.alignment}
+      <>
+        <Single
+          alignment={alignment}
+          onLongPress={onLongPress}
+          onPress={() => setViewerModalOpen(true)}
+          testID='image-attachment-single'
+          {...additionalTouchableProps}
         >
-          {images.slice(0, 4).map((image, i) => (
-            <ImageContainer
-              key={`gallery-item-${i}`}
-              length={images.length}
-              activeOpacity={0.8}
-              onPress={() => {
-                this.setState({
-                  viewerModalOpen: true,
-                  viewerModalImageIndex: i,
-                });
-              }}
-              onLongPress={this.props.onLongPress}
-              {...additionalTouchableProps}
-            >
-              {i === 3 && images.length > 4 ? (
-                <View
-                  style={{
-                    width: '100%',
-                    height: '100%',
-                  }}
-                >
-                  <Image
-                    style={{
-                      width: 100 + '%',
-                      height: 100 + '%',
-                    }}
-                    resizeMode='cover'
-                    source={{ uri: images[3].url }}
-                  />
-                  <View
-                    style={{
-                      position: 'absolute',
-                      height: '100%',
-                      width: '100%',
-                      display: 'flex',
-                      alignItems: 'center',
-                      justifyContent: 'center',
-                      backgroundColor: 'rgba(0,0,0,0.69)',
-                    }}
-                  >
-                    <Text
-                      style={{
-                        color: 'white',
-                        fontWeight: '700',
-                        fontSize: 22,
-                      }}
-                    >
-                      {' '}
-                      +{' '}
-                      {t('{{ imageCount }} more', {
-                        imageCount: images.length - 3,
-                      })}
-                    </Text>
-                  </View>
-                </View>
-              ) : (
-                <Image
-                  style={{
-                    width: 100 + '%',
-                    height: 100 + '%',
-                  }}
-                  resizeMode='cover'
-                  source={{ uri: image.url }}
-                />
-              )}
-            </ImageContainer>
-          ))}
-        </GalleryContainer>
+          <Image
+            resizeMode='cover'
+            source={{ uri: galleryImages[0].url }}
+            style={{ height: '100%', width: '100%' }}
```

Comment:
`flex: 1` not work here instead of height/width 100%?

Reply:
yep flex: 1 works for all of these, updating